### DISCUSSION TITLE
CycloneDX 1.6 isn't widely supported; consider producing 1.5 again instead

### DISCUSSION THREAD

**Discussion by candrews** [aquasecurity/trivy#7174] [Ideas]
### Description

Trivy 0.5.3 now produces CycloneDX 1.6 SBOM's: https://github.com/aquasecurity/trivy/issues/6902

However, tooling support for CycloneDX 1.6 is still spotty since it is so new. For example, cyclonedx-cli doesn't yet support it (https://github.com/aquasecurity/trivy/issues/6902) nor does GitLab (see https://gitlab.com/gitlab-org/gitlab/-/issues/472837).

### Desired Behavior

Can you please consider producing CycloneDX 1.5 again, or provide an option to specify the CycloneDX version for Trivy to produce?

### Actual Behavior

Trivy produces CycloneDX 1.6 SBOMs which are not (yet) widely supported.

### Reproduction Steps

```bash
n/a
```

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
n/a
```

### Operating System

Linux

### Version

```bash
Version: 0.53.0
```

### Checklist

- [ ] Run `trivy clean --all`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by knqyf263** [comment]
It's not our bug. I've changed the category.
Since they have released v1.6, it is more sound for their tools to support v1.6 rather than other tools downgrading the version. We would only reconsider if there is something special about the situation that they cannot handle.

**Reply by dirsigler** [reply]
What about allowing different specs similar as Syft does it ?
